src/main/java/uk/gov/hmcts/cmc/claimstore/repositories/DBCaseRepository.java

```diff
@@ -224,9 +224,12 @@ public void saveCaseEvent(String authorisation, Claim claim, CaseEvent caseEvent
     }
 
     @Override
-    public void saveClaimDocuments(String authorisation,
-                                   Long claimId,
-                                   ClaimDocumentCollection claimDocumentCollection) {
+    public Claim saveClaimDocuments(String authorisation,
+                                    Long claimId,
+                                    ClaimDocumentCollection claimDocumentCollection) {
         claimRepository.saveClaimDocuments(claimId, jsonMapper.toJson(claimDocumentCollection));
+        return claimRepository.getById(claimId).orElseThrow(() ->
```

Comment:
Why is saveClaimDocuments returning claim? I feel we should leave it to the service class to call this once after uploading all the document. Again its suggestion. It can reduce no of calls.

Reply:
This is so that we don't make multiple calls to CCD.